shotover/Cargo.toml

```diff
@@ -126,6 +126,11 @@ atoi = { version = "2.0.0", optional = true }
 fnv = "1.0.7"
 sasl = { version = "0.5.1", optional = true, default-features = false, features = ["scram"] }
 
+# Force C dependencies to be built in parallel e.g. ring has some C code it compiles with cc
+# Remove this if we no longer have cc in our dep tree.
+[build-dependencies]
+cc = { version = "1.0", features = ["parallel"] }
```

Comment:
This feature was previously enabled by a dependency of kafka-protocol, to retain this feature I've manually added it here.